### ISSUE TITLE
Is there is any way to resize columns size depends on the elements inside the columns ?

### ISSUE THREAD

**Issue by ThirupathiGovind** [kristoferjoseph/flexboxgrid#271]
### Briefly ( tweet size ) describe your issue and how you discovered it.

...

### Expected behavior

...

### Actual behavior

...

### Steps to reproduce behavior

* OS:
* Browser:
* code/fiddle/pen url:

**Comment by effone** [comment]
Auto column width is what you are looking for if I am getting your problem right ...
```
<div class="row">
    <div class="col-xs">
        Content ...
    </div>
</div>
```

**Comment by ThirupathiGovind** [comment]
@effone yes, it works. But it makes all columns equal size. I want to auto resize column width based on the text length. 

In my case, I have 7 columns. Each column has text with different size. I want that bigger text column should take larger size than the small text.

**Comment by effone** [comment]
Well you can define a column to fixed width or max width and the other's width as flexible (thats how sidebars work). Here is [a pen for example](https://codepen.io/ingomc/pen/BKqQzp).

 If all the column width are auto and non-fixed generally it will be distributed equally unless content on any one forces to maintain minimum size (you can check this reducing the size of viewport). It sounds weird but it happens. Look at the following image; the menu items are actually auto width columns and if you notice they are not equal in reduced size viewport because of their content (unless any word break or wrap is applied).

![menu](http://oi64.tinypic.com/32zqjol.jpg)
